beacon-chain/rpc/beacon_chain_server.go

```diff
@@ -156,18 +155,13 @@ func (bs *BeaconChainServer) ListBlocks(
 		}
 
 		if blk == nil {
-			return nil, status.Errorf(codes.NotFound, "block for root %#x does not exists in DB", q.Root)
-		}
-
-		token, err := strconv.Atoi(req.PageToken)
-		if err != nil {
-			return nil, status.Errorf(codes.Internal, "could not convert page token: %v", err)
+			return &ethpb.ListBlocksResponse{Blocks: []*ethpb.BeaconBlock{}, TotalSize: 0}, nil
 		}
 
 		return &ethpb.ListBlocksResponse{
 			Blocks:        []*ethpb.BeaconBlock{blk},
 			TotalSize:     1,
-			NextPageToken: strconv.Itoa(token + 1),
+			NextPageToken: req.PageToken,
```

Comment:
Can we just omit this field for a lookup by ID? It doesn't seem necessary to send back whatever the requester sent here.